pkg/template/api/types.go

```diff
@@ -16,6 +16,10 @@ type Template struct {
 	// Optional: Parameters is an array of Parameters used during the
 	// Template to Config transformation.
 	Parameters []Parameter `json:"parameters,omitempty"`
+
+	// Optional: ObjectLabels is a set of labels that are applied to every
+	// object during the Template to Config transformation
+	ObjectLabels map[string]string `json:"objectlabels,omitempty"`
```

Comment:
You can call it Labels externally and ObjectLabels internally.

----- Original Message -----
> > @@ -16,6 +16,10 @@ type Template struct {
> >  	// Optional: Parameters is an array of Parameters used during the
> >  	// Template to Config transformation.
> >  	Parameters []Parameter `json:"parameters,omitempty"`
> > +
> > +	// Optional: ObjectLabels is a set of labels that are applied to every
> > +	// object during the Template to Config transformation
> > +	ObjectLabels map[string]string `json:"objectlabels,omitempty"`
> 
> @smarterclayton @bparees @mfojtik - if the change suggested above is for
> v1beta2, then for now should it be called ObjectLabels? or ItemLabels? (I'm
> ok with either one). We can't call it Labels because it overrides the one in
> ObjectMetadata.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/982/files#r24821525